### DISCUSSION TITLE
Run SQANTI3 with annotated reference Genome but no annotated reference Transcriptome?

### DISCUSSION THREAD

**Discussion by KeSchus** [ConesaLab/SQANTI3#305] [Q&A] [answered]
I will do a comparison of alternatively spliced genes between 2 states and already have an annotated reference genome but no annotated reference Transcriptome. Is there a way to get around that in the QC step? 
Could I take the transcriptome of 1 state as a reference Transcriptome for the other state and vice versa?
Thank you in advance.

**Comment by TianYuan-Liu** [comment, marked as answer]
Hi KeSchus,

To better assist you, could you please specify which species you are using for your analysis?

Additionally, one approach to address the lack of a reference annotation is to use the Liftoff tool. Liftoff can help you map annotations from a closely related species' reference genome to your species of interest. You can find more details about Liftoff in this article: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8289374/.

As for your question, using the transcriptome of one state as a reference for the other and vice versa could introduce bias and complicate your analysis. It’s generally more reliable to use a properly annotated reference if possible.

Tian

**Comment by KeSchus** [comment]
Hi Tian,

thank you for your quick response and input. I am comparing differential Splicing between the symbiotic and aposymbiotic states in the anemone Aiptaisia using Oxford Nanopore longreads. 

I'm still a little confused. I do have an annotated genome and my own Transcriptome as well (I did 3 assemblies using different tools). But there is no accessible annotated reference Transcriptome. I thought I just might take raw sequencing data from NCBI and do it myself just so I have something as input for the QC step.
I proceeded using SUPPA2 first but am planning to still do the SQANTI analysis afterwards and compare the outcomes.

And yea your point with introducing a bias makes sense, so I'll just have to figure out a way around that.
Thank you!

cheers,
Kevin

Am Mittwoch, Juni 05, 2024 17:23 CEST, schrieb Tianyuan Liu ***@***.***>:
